import/openrailwaymap.lua

```diff
@@ -604,6 +605,7 @@ function osm2pgsql.process_way(object)
       preserved_railway = tags['preserved:railway'],
       train_protection = railway_train_protection,
       train_protection_rank = railway_train_protection_rank,
+      operator = tags['operator'],
```

Comment:
The operator tag can have a semicolon delimited list of operators. It isn't guaranteed to simply be a single operator.

It's a frequent thing in North America, where the primary operator/owner also leases access to other companies

Reply:
Nice, I will turn it into an array, so it can be displayed comma-separated in the UI!

Reply:
Done in d9b0b5eff39547652069ac5bbb861580cdbb8c2b